### DISCUSSION TITLE
[Question] Number of domains per identity provider

### DISCUSSION THREAD

**Discussion by SangI762** [sventorben/keycloak-home-idp-discovery#238] [General]
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Is your feature related to a problem? Please describe.

Hello, it's not an issue or feature request, I'm just wondering how many domains I could use on single idp.
I have use-case where I need to have let's say 200-300 domains. How will it handle? Maybe you did some performance tests or have clue what will happen in that case?

**Comment by sventorben** [comment]
Hello @SangI762,

no, I did not test it.
What causes >200 domains per identity provider?

Regards
Sven-Torben

**Reply by pcMarcos** [reply]
hey @sventorben, how are you?

I would like if there's any limit of domains. In my case, I don't need thousands of domains, but I'll need at least 10, maybe more. 

Should I encounter any issue with that number?

**Reply by sventorben** [reply]
Hey @pcMarcos 

I don't see a reason why 10 domains per IdP should be an issue unless you have a high number of IdPs as well.

Best,
Sven-Torben
